### PR TITLE
add Arch Linux instructions to README.md

### PR DESCRIPTION
**Pull request by invsblduck**
And other touch ups.

### CODE REVIEW

**Comment by invsblduck** [comment]
These modifications will help your new users get up and running faster!  The `--verbose` option to `vagrant plugin install` is really nice to see the `./configure` output etc. while it's building.

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Next, you must have [Vagrant installed](http://docs.vagrantup.com/v2/installatio
  Now you're ready to install vagrant-libvirt using standard [Vagrant plugin](http://docs.vagrantup.com/v2/plugins/usage.html) installation methods.
 
 ```
-$ vagrant plugin install vagrant-libvirt
+$ vagrant plugin install vagrant-libvirt --verbose
```
Comment (pronix) [inline README.md]: this changes not required

File: README.md [hunk 2]
```diff
@@ -54,8 +54,15 @@ In RedHat, Centos, Fedora, ...
 # yum install libxslt-devel libxml2-devel libvirt-devel libguestfs-tools-c
 ```
 
-If have problem with installation - check your linker. It should be ld.gold:
```
Comment (pronix) [inline README.md]: do not change anything about linker

File: README.md [hunk 3]
```diff
@@ -54,8 +54,15 @@ In RedHat, Centos, Fedora, ...
...
+In Arch ...
```
Comment (pronix) [inline README.md]: installation for arch you can define in specific paragraph

**Comment by invsblduck** [comment]
lol, ok bro... c ya later kook.

**Pull request closed without merging**